### DISCUSSION TITLE
Why is Baram producing NAN in files?

### DISCUSSION THREAD

**Discussion by OttersMeep** [nextfoam/baram#58] [Q&A]
I'm running macOS Sonoma on Apple Silicon (m2 Pro)
[PlaneFinished.zip](https://github.com/nextfoam/baram/files/14454852/PlaneFinished.zip)
[stderr.log](https://github.com/nextfoam/baram/files/14454854/stderr.log)
[stdout.log](https://github.com/nextfoam/baram/files/14454855/stdout.log)

**Comment by kdg5424** [comment]
Hi @OttersMeep this is Donggyu.
Thanks for your interest in BARAM.
It looks like the solver diverged.
Maybe the reason why the solve diverged is 'small domain', 'insufficient mesh'
so I recommend you these solutions.

1. Extend the simulation domain
    The domain looks like small. I recommend you why don't you extend the domain.
    I recommend you at least extend 10~20 times than current domain.

2. Regenerate the mesh
    The quality and the quantity of mesh is insufficient. So I suggest you regenrate the mesh. The below picture is the reference.

![image](https://github.com/nextfoam/baram/assets/133930861/c901c8ac-2833-4bea-8d5c-caa5086b7b30)

After that, if the problem happens again, don't hesitate to ask me.

Thanks
